File: eSpaCenter.Models/Uplata.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eSpaCenter.Models
{
    public class Uplata
    {
        public int UplataID { get; set; }
        public double Iznos { get; set; }
        public DateTime DatumTransakcije { get; set; }
        public string BrojTransakcije { get; set; }
    }
}

File: UI/espacenter_admin/lib/models/uplata.dart
import 'package:json_annotation/json_annotation.dart';

part 'uplata.g.dart';

@JsonSerializable()
class Uplata{
  int? uplataID;
  double? iznos;
  DateTime? datumTransakcije;
  String? brojTransakcije;
  int? korisnikID;

  Uplata(this.uplataID,this.iznos,this.datumTransakcije,this.brojTransakcije,this.korisnikID);

  factory Uplata.fromJson(Map<String, dynamic> json) => _$UplataFromJson(json);

    Map<String, dynamic> toJson() => _$UplataToJson(this);
}